src/SFA.DAS.CaptureTheFlag.Web/Views/Settings/Index.cshtml

```diff
@@ -0,0 +1,45 @@
+﻿@model SettingsModel
+@{
+    Layout = "_Layout";
+}
+
+<h1>Account Settings</h1>
+<br />
+<br />
+<br />
+
+<h1>Game Settings</h1>
+
+@using (Html.BeginForm("Post", "Settings", FormMethod.Post))
+{
+    <h3>Music Volume</h3>
+    <span>
+        @Html.DropDownListFor(model => model.MusicVolume, Enumerable.Range(1, 5)
```

Comment:
The range numbers would also be part of a configuration table in the database eventually